docs/examples/promise/03_listen_to_balance_change/index.js

```diff
@@ -24,7 +24,7 @@ async function main () {
   // Here we subscribe to any balance changes and update the on-screen value
   api.query.genericAsset.freeBalance(CENNZ, Alice, (current) => {
     // Calculate the delta
-    const change = current.sub(previous);
+    const change = previous.sub(current);
```

Comment:
why this change 😅

Reply:
I can leave it what it was before. Actually.. I tried to transfer balance from Alice to Bob.. so
previous balance > current balance and it would show in negative..
but it can also be the other way around, when someone is trying to send some balance to Alice..
I will revert this change

Reply:
updated this to check if balance is sent or transferred from Alice's account..